modular_skyrat/modules/customization/modules/clothing/glasses/hud.dm

```diff
@@ -3,11 +3,19 @@
 	desc = "A simple HUD designed to interface with optical nerves of a lost eye. This one seems busted."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/glasses.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/eyes.dmi'
-	icon_state = "hudpatch_R"
+	icon_state = "hudpatch"
 	base_icon_state = "hudpatch"
 	inhand_icon_state = "sunhudmed"
 	uses_advanced_reskins = TRUE
 	can_switch_eye = TRUE	//See modular_skyrat\modules\customization\modules\clothing\glasses\glasses.dm
```

Comment:
This should be able to be removed now that our system's gutted

Reply:
Ditto.